beta/docs/recipes/next/integrate.md

```diff
@@ -0,0 +1,93 @@
+---
+id: nextjs
+title: Integrate next js with effector
+---
+
+To do this, we will use the native [fork](https://effector.dev/docs/api/effector/fork) method.
+In the __app.tsx_ file, we create our own [Scope](https://effector.dev/docs/api/effector/scope) and wrap the application in a provider from _effector-react/scope_
+
+```js
+import { AppProps } from 'next/app';
+import { FC } from 'react';
+import { Provider } from 'effector-react/scope';
+import { fork, Scope, serialize } from 'effector';
+
+let clientScope: Scope;
+
+const App: FC<AppProps<{ initialState }>> = ({ Component, pageProps }) => {
+    const scope = fork({
+        values: {
+            ...(clientScope && serialize(clientScope)),
+            ...pageProps.initialState,
+        },
+    });
+
+    if (typeof window !== 'undefined') {
+        clientScope = scope;
+    }
+
+    return (
+        <Provider
+            key={scope?.graphite?.id || '0'}
+            value={scope}
+        >
+            <Component {...pageProps} />
+        </Provider>
+    );
+};
+
+export default App;
+```
+
+Next, you need to add _.babelrc_ to the project root with the following content:
+___
+.babelrc
+___
+
+```js
+{
+    "plugins": [
+        [
+            "effector/babel-plugin", {
+            "reactSsr": true,
+        }
+        ]
+    ]
+}
+```
+You also need to add an alias to _next.config.js_ to avoid build errors, when under the hood it imports, then cjs then mjs files.
+```js
```

Comment:
Please add an empty line between Code block and text. Most markdown plugins don't like it.

For example, here we have broken highlight

![image](https://user-images.githubusercontent.com/5620073/209830559-20676798-926a-40e0-90fd-c94234653f07.png)

Reply:
done